### PR TITLE
remove iframely logic from SDKs

### PR DESCRIPTION
**Pull request by steve8708**
that way we can more easily update this and rotate keys

this is now overriden in our web app and not needed int he SDK source code

### CODE REVIEW

File: packages/react/src/blocks/Embed.tsx [hunk 1]
```diff
@@ -109,6 +110,8 @@ export const Embed = withBuilder(EmbedComponent, {
           options.delete('content');
         }
       },
+=======
+>>>>>>> Stashed changes
```
Comment (samijaber) [inline packages/react/src/blocks/Embed.tsx:114]: euh oh!

Comment (steve8708) [reply packages/react/src/blocks/Embed.tsx]: i swear i know how to code

File: packages/sdks/src/blocks/embed/component-info.ts [hunk 2]
```diff
@@ -20,35 +20,6 @@ export const componentInfo: ComponentInfo = {
       required: true,
       defaultValue: '',
       helperText: 'e.g. enter a youtube url, google map, etc',
-      onChange: (options: Map<string, any>): void | Promise<void> => {
-        const url = options.get('url');
-        if (url) {
-          options.set('content', 'Loading...');
-
-          const apiKey = _iframelyApiKey;
```
Comment (samijaber) [inline packages/sdks/src/blocks/embed/component-info.ts:28]: I think you can also delete the `declare const _iframelyApiKey: string;` in these files as a result, no?

Comment (steve8708) [reply packages/sdks/src/blocks/embed/component-info.ts:28]: tough crowd

**Review by samijaber — APPROVED** [review]

**Pull request merged**